src/enclave/Enclave/ExpressionEvaluation.h

```diff
@@ -151,6 +151,13 @@ flatbuffers::Offset<tuix::Field> eval_binary_comparison(
         static_cast<const tuix::DoubleField *>(right->value())->value());
       break;
     }
+    case tuix::FieldUnion_DateField:
```

Comment:
This change seems unrelated to the switch to OpenEnclave. It would be better to put it in a different PR.

Reply:
Moved to #94.